src/Domain/ContentRenderer/CodeRenderer.php

```diff
@@ -56,6 +57,10 @@ private function getWrapperAttributes( string $contentUrl ): array {
 
 		$attributes['class'] = $this->getWrapperClasses();
 
+		if ( !empty( $this->showSpecificLines ) ) {
+			$attributes['data-show-lines'] = $this->showSpecificLines;
```

Comment:
It would be nice if we could normalize and sanitize this value so that the Javascript can assume it's getting valid values/ranges.

So far it doesn't look like it causes visual problems when I use this:
```
{{#bitbucket:http://bitbucket:7990/projects/TEST/repos/test/browse/test.php|specificLines=1-3,foo-bar,baz}}
```